tests/topotato/requirements.txt

```diff
@@ -0,0 +1,7 @@
+graphviz==0.20; python_version >= "3.8"
+Jinja2==3.1.2; python_version >= "3.8"
+lxml==4.8.0; python_version >= "3.8"
+MarkupSafe==2.0.1; python_version >= "3.8"
+pytest==6.2.4; python_version >= "3.8"
```

Comment:
If I remember correctly this should be `>=6.1 <7`
(I'm working on fixing pytest 7.x compatibility, but it's not there yet.)

Reply:
updated